charts/airflow/values.yaml

```diff
@@ -316,6 +316,12 @@ airflow:
     ##
     extraPipPackages: []
 
+    ## extra init containers to run in the kubernetesPodTemplate Pods
```

Comment:
nit: can you make this consistent with the other docscrings in the section:

```yaml
## extra init-containers to run in the Pod template
```

Reply:
ah makes sense. I fixed it